src/components/MAPI/organizations/AccessControl/AccessControlRoleSubjects.tsx

```diff
@@ -246,6 +258,14 @@ const AccessControlRoleSubjects: React.FC<IAccessControlRoleSubjectsProps> = ({
     };
   }, [roleName]);
 
+  const client = useHttpClient();
+  const auth = useAuthProvider();
+
+  const { data: serviceAccountSuggestions } = useSWR<string[], GenericResponse>(
```

Comment:
We ignore the error. We just won't display any suggestions if fetching service accounts fails.

Reply:
Shouldn't we have some sort metrics/logging for that?

BTW where is that error?

Reply:
This line fetches the service accounts from the K8s API, so they could be shown as suggestions in the service account binding form. An error might only occur if the K8s API is unreachable (which might be a problem on the user's side, or on the ingress side). I don't believe that `happa` should report this error, as it isn't really its fault.

If we were to display the error to the user, we could destructure it from our request controller object, and place it somewhere in the UI (in a flash message, for example).

```suggestion
  const { data: serviceAccountSuggestions, error } = useSWR<string[], GenericResponse>(
```

I've decided to ignore the error if it occured because the only difference in UX would be that the user would get no suggestions for existing service accounts. Everything else would work as expected.

However, if you think it's important to display the error, I see no problem in doing it 👍🏻

Reply:
Jeez I keep forgetting the request comes from the browser.